File: spring-cloud-config/config-jdbc-server/src/main/java/com/livk/server/ConfigJdbcServer.java
package com.livk.server;

import com.livk.commons.SpringLauncher;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.config.server.EnableConfigServer;

/**
 * <p>
 * ConfigServer
 * </p>
 *
 * @author livk
 */
@EnableConfigServer
@SpringBootApplication
public class ConfigJdbcServer {

    public static void main(String[] args) {
        SpringLauncher.run(args);
    }

}

File: spring-cloud-config/config-jdbc-server/src/main/kotlin/com/livk/server/ConfigJdbcServer.kt
package com.livk.server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.config.server.EnableConfigServer

/**
 *
 *
 * ConfigServer
 *
 *
 * @author livk
 */
@EnableConfigServer
@SpringBootApplication
open class ConfigJdbcServer

fun main(args: Array<String>) {
    runApplication<ConfigJdbcServer>(*args)
}
